core/frontend/src/components/video-manager/VideoStream.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-card
+    width="100%"
+    class="pa-0 my-4"
+  >
+    <div class="d-flex flex-no-wrap align-center justify-space-between">
+      <div>
+        <v-card-title>{{ stream.video_and_stream.name }}</v-card-title>
+        <v-card-subtitle>{{ settings_summary }}</v-card-subtitle>
+      </div>
+      <div>
+        <v-btn
+          class="ma-2"
+          @click="openDetailsDialog"
+        >
+          <v-icon>mdi-eye</v-icon>
+          View details
```

Comment:
I think that this defails card is not necessary, we can have all the information on the main card:
![2021-09-22_11-38](https://user-images.githubusercontent.com/1215497/134364694-f80f7552-f4ea-4bfc-a23a-d3f4ba25a57a.png)